src/Passes/docs/src/DeveloperGuide/CodeQuality.md

```diff
@@ -0,0 +1,167 @@
+# Code quality and continuous integration
+
+## Using the manage tool
+
+Before making a pull request with changes to this library, please ensure that style checks passes, that the code compiles,
+unit test passes and that there are no erros found by the static analyser.
+
+To setup the CI environment, run following commands
+
+```sh
+source develop.env
```

Comment:
FYI: I fail to find `develop.env` or `*.env`.

Reply:
Thanks - shouldn't be there.